pypose/__init__.py

```diff
@@ -3,13 +3,15 @@
 from ._version import __version__
 
 from .lietensor import LieTensor, Parameter, SO3, so3, SE3, se3, Sim3, sim3, RxSO3, rxso3
+from .lietensor import SO2, so2, SE2, se2
```

Comment:
Not sure about the convention here. Adding these imports to the previous line, makes it too long. Maybe keeping `LieTensor` and `Parameter` on an extra line is preferred?

Reply:
yes, that is a better way.